server/api/tours.js

```diff
@@ -8,7 +8,10 @@ router.get('/:tourId', async(req, res, next) => {
     const tourId = req.params.tourId;
     const tourSnapshot = await db.ref(`/tours/${tourId}`).once('value');
     const tour = tourSnapshot.val();
-
+    if(!tour){
```

Comment:
Here actually @Peeeeeng , why don't we just return 404 code, instead of returning an empty object?

Reply:
I set my code to response to an object that returned. I guess I could make the code response to 404 too......after I finished what I'm working on right now